package.json

```diff
@@ -47,87 +44,87 @@
+        "common-fix-lint": "pnpm -F @lightdash/common fix-lint",
+        "common-fix-format": "pnpm -F @lightdash/common fix-format",
+        "common-test": "pnpm -F @lightdash/common test",
+        "backend-dev": "pnpm -F backend dev",
+        "backend-build": "pnpm -F backend build",
+        "backend-start": "pnpm -F backend start",
+        "backend-lint": "pnpm -F backend lint",
+        "backend-format": "pnpm -F backend format",
+        "backend-fix-lint": "pnpm -F backend fix-lint",
+        "backend-fix-format": "pnpm -F backend fix-format",
+        "backend-test": "pnpm -F backend test",
+        "frontend-dev": "pnpm -F frontend dev",
+        "frontend-build": "pnpm -F frontend build",
+        "frontend-lint": "pnpm -F frontend lint",
+        "frontend-format": "pnpm -F frontend format",
+        "frontend-fix-lint": "pnpm -F frontend fix-lint",
+        "frontend-fix-format": "pnpm -F frontend fix-format",
+        "frontend-test": "pnpm -F frontend test",
+        "e2e-open": "pnpm -F e2e cypress:open",
+        "e2e-run": "pnpm -F e2e cypress:run",
+        "e2e-lint": "pnpm -F e2e lint",
+        "e2e-format": "pnpm -F e2e format",
+        "e2e-fix-lint": "pnpm -F e2e fix-lint",
+        "e2e-fix-format": "pnpm -F e2e fix-format",
+        "cli-test": "pnpm -F @lightdash/cli test",
+        "cli-lint": "pnpm -F @lightdash/cli lint",
+        "cli-format": "pnpm -F @lightdash/cli format",
+        "cli-fix-lint": "pnpm -F @lightdash/cli fix-lint",
+        "cli-fix-format": "pnpm -F @lightdash/cli fix-format",
+        "cli-build": "pnpm -F @lightdash/cli build",
+        "warehouses-dev": "pnpm -F @lightdash/warehouses dev",
+        "warehouses-lint": "pnpm -F @lightdash/warehouses lint",
+        "warehouses-format": "pnpm -F @lightdash/warehouses format",
+        "warehouses-fix-lint": "pnpm -F @lightdash/warehouses fix-lint",
+        "warehouses-fix-format": "pnpm -F @lightdash/warehouses fix-format",
+        "warehouses-test": "pnpm -F @lightdash/warehouses test",
+        "warehouses-build": "pnpm -F @lightdash/warehouses build",
+        "lint": "pnpm run common-lint && pnpm run backend-lint && pnpm run frontend-lint && pnpm run e2e-lint && pnpm run cli-lint",
+        "format": "pnpm run common-format && pnpm run backend-format && pnpm run frontend-format && pnpm run e2e-format && pnpm run cli-format",
+        "fix-lint": "pnpm run common-fix-lint && pnpm run backend-fix-lint && pnpm run frontend-fix-lint && pnpm run e2e-fix-lint && pnpm run cli-fix-lint",
+        "fix-format": "pnpm run common-fix-format && pnpm run backend-fix-format && pnpm run frontend-fix-format && pnpm run e2e-fix-format && pnpm run cli-fix-format",
+        "test": "pnpm run common-test && pnpm run warehouses-test && pnpm run cli-test && pnpm run backend-test && pnpm run frontend-test",
+        "dev": "pnpm -r run dev --parallel -F @lightdash/common -F @lightdash/warehouses -F backend -F frontend",
```

Comment:
```suggestion
        "dev": "pnpm -r --parallel --stream --filter "@lightdash/common" --filter "@lightdash/warehouses" --filter "backend" --filter "frontend" run dev",
```

Reply:
--filter flags before run dev so that nothing extra is passed to the underlying commands